FrontEnd/src/Components/HeroSection.jsx

```diff
@@ -1,12 +1,18 @@
+import { useState } from "react";
 import { Text, Box, Flex, Button, Link, Image } from "@chakra-ui/react";
 import { MdOpenInNew } from "react-icons/md";
 import { TypeAnimation } from "react-type-animation";
 import heroimage from "./assets/heroimage.svg";
 import ModalForm from "./ModalForm";
 
 const HeroSection = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
`useDisclosure` hook from chakraUI can be used instead of implementing the functionality manually. Reference - https://chakra-ui.com/docs/hooks/use-disclosure

Reply:
Will refer to it!